lib/chef_zero/server.rb

```diff
@@ -544,7 +546,13 @@ def open_source_endpoints
         [ "/organizations/*/nodes", RestListEndpoint.new(self) ],
         [ "/organizations/*/nodes/*", NodeEndpoint.new(self) ],
         [ "/organizations/*/nodes/*/_identifiers", NodeIdentifiersEndpoint.new(self) ],
-        [ "/organizations/*/policies/*/*", PoliciesEndpoint.new(self) ],
+        [ "/organizations/*/policies", OrganizationPoliciesEndpoint.new(self) ],
+        [ "/organizations/*/policies/*", OrganizationPoliciesEndpoint.new(self) ],
+        [ "/organizations/*/policies/*/revisions", OrganizationPoliciesEndpoint.new(self) ],
+        [ "/organizations/*/policies/*/revisions/*", OrganizationPoliciesEndpoint.new(self) ],
+        [ "/organizations/*/policy_groups", OrganizationPolicyGroupsEndpoint.new(self) ],
+        [ "/organizations/*/policy_groups/*", OrganizationPolicyGroupsEndpoint.new(self) ],
+        [ "/organizations/*/policy_groups/*/policies/*", OrganizationPolicyGroupsEndpoint.new(self) ],
```

Comment:
It seems like `/organizations/*/policies/*/*` would still match the 4 endpoints that got expanded - can we just continue using that matcher?  Or does it change how the params are passed into the Endpoint class?